files/image_config/interfaces/interfaces.j2

```diff
@@ -5,6 +5,11 @@
 # file: /etc/network/interfaces
 #
 {% endblock banner %}
+{% if (MGMT_VRF_CONFIG) and (MGMT_VRF_CONFIG['vrf_global']['mgmtVrfEnabled'] == "true") %}
+auto mgmt
+iface mgmt
+    vrf-table 1001
```

Comment:
this is conflict with sonic-swss vrf assignment. Can you use 1000 instead?

https://github.com/Azure/sonic-swss/blob/master/cfgmgr/vrfmgr.cpp#L11

Reply:
Will use 1000